### ISSUE TITLE
Repeating Copilot Chat Answer

### ISSUE THREAD

**Issue by AVMiraga** [microsoft/vscode-copilot-release#897]
I didn't find an active or duplicate issue here.

- VS Code Version: 1.87.0 insider (didn't try in VS Code)
- OS Version: Win 11
- Logs: Prompt and Answer:
Prompt:
`Explain what these options mean?`
Answer:
`I'm sorry, but it seems like there might be a misunderstanding. "Active selection" is not clear in this context. Could you please provide more details or clarify your question? I'm here to help!`
===============
Prompt:
`is it possible to replace myOptions.PermitLimit with digit?` (Selection Made in editor (Used 1 reference))
Answer:
`I'm sorry, but it seems like there might be a misunderstanding. "Active selection" is not clear in this context. Could you please provide more details or clarify your question? I'm here to help!`

I didn't even say active selection, why is it happening?

Also, I was using the release version until today, and there was a problem `error from provider: Cannot read properties of undefined (reading 'map')` which users solved by switching from release to pre-release.

TLDR:
pre-release gives an error.
release gives the same answer no matter what you ask.

Steps to Reproduce:

1. Open Copilot Chat
2. Type Something
3. Get Answer
4. Try to explain it further more
5. Get Same Answer

**Comment by AVMiraga** [comment]
Seems to be fixed with `v0.13.2024021501 (pre-release)`.

**Comment by ihongda** [comment]
I encountered the same problem.

**Comment by roblourens** [comment]
This sounds like an issue with the way we build context for chat (which includes your selection) but I don't know what would cause this issue. I haven't seen it.

> On the other hand "New Chat" Fixes the problem.

Thanks for the context.

I would really appreciate if you share the full chat next time this happens, eg by right clicking > "Copy All", if you are able to share it. Or any other context, like whether you had a selection or open editors.

**Comment by AVMiraga** [comment]
> I would really appreciate if you share the full chat next time this happens, eg by right clicking > "Copy All", if you are able to share it. Or any other context, like whether you had a selection or open editors.

Sure, but I'm afraid some chat parts contain private data, is there any way for me to send it to you privately? Also, how can I share selections or open editors?

**Comment by roblourens** [comment]
Feel free to email it to the address in my profile

**Comment by AVMiraga** [comment]
Sorry for the Late Reply, the problem is not persisting anymore, It seems to be fixed. If no one encountered the problem, the thread can be closed. Thanks :)
